js/fileupload.js

```diff
@@ -64,7 +64,13 @@ function uploadFile(file, editor, input_name) {
       },
 
       error: function (request) {
-         alert(request.responseText);
+         // If this is an error on the return
+         if (request.responseText !== undefined) {
+            alert(request.responseText);
+         } else {
+            // Error before sending request #3866
+            alert(request.statusText);
```

Comment:
i don't have in mind what text this could return, but i think alert (sending status to final user) it is a bad idea.
Doing a `console.error(...)` could be better

Reply:
@orthagh In case of css error (as in this case for IE), the user have an empty pop-up that is showing.
Imho, it's better to have a "technical" error message than a empty box.

For example, if you test on IE11, you will have "Error : Access Refused" that is showing

Regards